helpers/content.go

```diff
@@ -359,6 +429,15 @@ func (c *RenderingContext) getConfig() *Blackfriday {
 	return c.Config
 }
 
+func (c *RenderingContext) getMmarkConfig() *Mmark {
+	c.configInit.Do(func() {
+		if c.MmarkConfig == nil {
+			c.MmarkConfig = NewMmark()
```

Comment:
Indeed, I was feeling bit uneasy about that getMmarkConfig() thingy but didn't know any better.

Thank you Bjørn for your advice!  I will try to understand your method better and work on it in the coming week.